Gemfile

```diff
@@ -4,3 +4,5 @@ gem 'minitest'
 gem 'rubocop', '0.36.0'
 gem 'simplecov'
 gem 'rake'
+gem 'values'
```

Comment:
I am always hesitant to add new dependencies to a project, do these gems provide anything that is essential and we can't handle ourselves?